src/modules/rangeproof/tests_impl.h

```diff
@@ -541,8 +541,95 @@ static void test_rangeproof(void) {
     }
 }
 
+static void test_single_value_proof(uint64_t val) {
+    unsigned char proof[5000];
+    secp256k1_pedersen_commitment commit;
+    unsigned char blind[32];
+    unsigned char blind_out[32];
+    unsigned char nonce[32];
+    const unsigned char message[1] = " "; /* no message will fit into a single-value proof */
+    unsigned char message_out[sizeof(proof)] = { 0 };
+    size_t plen = sizeof(proof);
+    size_t min_val_out = 0;
+    size_t max_val_out = 0;
+    size_t val_out = 0;
```

Comment:
```suggestion
    uint64_t min_val_out = 0;
    uint64_t max_val_out = 0;
    uint64_t val_out = 0;
```

Reply:
Thanks. I'm disappointed that my local gcc didn't catch that.